datasets/per_sent/per_sent.py

```diff
@@ -0,0 +1,147 @@
+            names=["", "Negative", "Neutral", "Positive"]
+        )
+        features = datasets.Features(
+            {
+                "DOCUMENT_INDEX": datasets.Value("int64"),
+                "TITLE": datasets.Value("string"),
+                "TARGET_ENTITY": datasets.Value("string"),
+                "DOCUMENT": datasets.Value("string"),
+                "MASKED_DOCUMENT": datasets.Value("string"),
+                "TRUE_SENTIMENT": label,
+                "Paragraph0": label,
+                "Paragraph1": label,
+                "Paragraph2": label,
+                "Paragraph3": label,
+                "Paragraph4": label,
+                "Paragraph5": label,
+                "Paragraph6": label,
+                "Paragraph7": label,
+                "Paragraph8": label,
+                "Paragraph9": label,
+                "Paragraph10": label,
+                "Paragraph11": label,
+                "Paragraph12": label,
+                "Paragraph13": label,
+                "Paragraph14": label,
+                "Paragraph15": label,
+            }
+        )
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage="https://stonybrooknlp.github.io/PerSenT",
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        train_path = dl_manager.download(_URLs["train"])
+        dev_path = dl_manager.download(_URLs["dev"])
+        test_fixed_path = dl_manager.download(_URLs["test_fixed"])
+        test_random_path = dl_manager.download(_URLs["test_random"])
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={
+                    "filepath": train_path,
+                    "split": "train",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=NamedSplit("test_random"),
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={"filepath": test_random_path, "split": "test_random"},
+            ),
+            datasets.SplitGenerator(
+                name=NamedSplit("test_fixed"),
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={"filepath": test_fixed_path, "split": "test_fixed"},
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={
+                    "filepath": dev_path,
+                    "split": "dev",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath, split):
+        """ Yields examples. """
+
+        with open(filepath) as f:
```

Comment:
could you add the encoding here ? for example `encoding="utf-8"`

Reply:
Removed the extraneous zip file and fixed encoding.  CI failing on [run_dataset_script_tests_pyarrow_1_WIN](https://circleci.com/gh/huggingface/datasets/7347?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)?